onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -390,17 +393,24 @@ QNNExecutionProvider::QNNExecutionProvider(const ProviderOptions& provider_optio
                           << "handles the graph I/O quantization/dequantization.";
   }
 
-  qnn_backend_manager_ = std::make_unique<qnn::QnnBackendManager>(
-      std::move(backend_path),
-      profiling_level_etw,
-      profiling_level,
-      std::move(profiling_file_path),
-      context_priority,
-      std::move(qnn_saver_path),
-      device_id_,
-      htp_arch,
-      soc_model,
-      enable_htp_weight_sharing);
+  static const std::string QNN_HTP_SHARED_MEMORY_ALLOCATOR_ENABLED = "enable_htp_shared_memory_allocator";
```

Comment:
Should this be more user visible?

Reply:
documented in onnxruntime_c_api.h. I can also document it in the gh-pages branch after this PR.